analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ProgramStateTest.cs

```diff
@@ -18,6 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
This file should also get simplified a lot once it's a property

Reply:
I'd like to keep at least two fields, to make sure the removal of the constraints is done the right way. I also like to place at least two constraints on each symbol for the same reason. So the test case gets only slightly simpler.

Reply:
Let's use the `LockConstraint` then as 2nd. It's not worth overengineering the test framework for this